### ISSUE TITLE
HW1 solutions Discretized Mixture of Logistics Parameter initialization confusion

### ISSUE THREAD

**Issue by cangozpi** [rll/deepul#14]
Hi,
I was going through hw1_solutions at https://github.com/rll/deepul/blob/master/homeworks/solutions/hw1_solutions.ipynb and I got confused at the _Discretized Mixture of Logistics_ implementation.
In the \__init\__ function of the class, the parameters(means, log_scales, logits) are all initialized in a different way. I couldn't understand how those initializations were decided. For example, why don't we declare all of these with torch.randn() ? 
![image](https://user-images.githubusercontent.com/54705314/158199969-ecbb7c01-e364-4756-a274-decfce5487dc.png)
I will appreaciate any help. Thank you for your time.

**Comment by wilson1yan** [comment]
I believe the initializations made it faster to train. `logits` are mixture parameters and initialized to be uniform. `means` are initialized to be uniformly distributed across the entire range of possible values (e.g. evenly spaced bins from 0-255 if we're doing rgb values). `log_scales` are just initialized randomly, but could also be initialized to unit variance. Overall, I think the `means` initialization helped the most and the `logits` and `log_scales` initializations could be zeros or randn.
